playbooks/roles/airship-deploy-osh/tasks/main.yml

```diff
@@ -86,6 +86,13 @@
     - install
     - always
 
+- name: Make sure shipyard can read the pegleg output
+  become: yes
```

Comment:
what permission was this set to when shipyard can't read it?

Reply:
owned by root:root and mode 0640